### DISCUSSION TITLE
Nextclooud AIO for local ?

### DISCUSSION THREAD

**Discussion by BoBBer446** [nextcloud/all-in-one#4684] [Questions] [answered]
I have no tld and there is no lets encrypt certificate.

Possible to use with local server about ip ?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/blob/main/local-instance.md

**Comment by BoBBer446** [comment]
Oh dear, I find that quite complicated. All solutions are somehow strange, why is it so essential that a certificate is used which is valid in the local network 

thx for response
